doctor_visits/delphi_doctor_visits/download_claims_ftp_files.py

```diff
@@ -51,9 +51,13 @@ def change_date_format(name):
     name = '_'.join(split_name)
     return name
 
-def download(ftp_credentials, out_path, logger):
+def download(ftp_credentials, out_path, logger, issue=None):
```

Comment:
As a general thought: can we introduce using typing for the function parameters? I checked that typing was introduced as a part of the standard library at 3.5 so should be able to use it for this repo. As a newcomer it would be helpful to know the type.

Reply:
@aysim319 can you turn this request into an issue?